docker/reqmgr2ms-unmerged/monitor.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# start process exporter
+for p in "config-monitor" "config-output" "config-transferor" "config-ruleCleaner"; do
```

Comment:
It is unclear to me if this applies to this service. Does it have any of this configs? If not, the process monitor will not start, but if it is then the port may be identical to what is inside of this block for other services. Since you blindly copy-and-paste this script I want to make sure that you are not forgetting anything specific to this particular service.

Reply:
that is a correct statement - none of those configs are present for the serivce itself while the one that is for the current service `config-unmerged` is  actually missing in this list. fixing this one too.